server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/camel/CamelProcessor.java

```diff
@@ -70,7 +68,7 @@ public void process(Exchange exchange) throws Exception {
         Mail mail = exchange.getIn().getBody(Mail.class);
         long start = System.currentTimeMillis();
         TimeMetric timeMetric = metricFactory.timer(mailet.getClass().getSimpleName());
-        MessagingException ex = null;
```

Comment:
How is the `ex` variable usefull?

Can't we get a ride of it?

Reply:
The `ex` variable is used in the `finally` clause, so we can't get rid of it.

Reply:
Ok, got it. Let's live with it...